SU-SWS/decanter
# 305 Z-Index Map and Function

# READY FOR REVIEW

# Summary
- Making a z-index map and a function to easily access the map
- Obviously there is a lot more that could be done to extend this. The map provided here is based on existing elements with z-index set, but not with the same values (since 9999 should not be necessary here.)
- This PR does NOT include swapping out the existing values, but if this looks good, I could do that next.

# Needed By (Date)
- End of sprint would be nice 

# Urgency
- Not very

# Steps to Test

1. Use function z($level) somewhere and see it pull the correct value from the map or error out if you use a bad name


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
